src/bootstrap-table.js

```diff
@@ -1585,16 +1585,16 @@ class BootstrapTable {
       }
 
       // handle class, style, id, rowspan, colspan and title of td
-      for (const item of ['class', 'style', 'id', 'rowspan', 'colspan', 'title']) {
-        const value = item[`_${field}_${item}`]
+      for (const attr of ['class', 'style', 'id', 'rowspan', 'colspan', 'title']) {
+        const value = item[`_${field}_${attr}`]
```

Comment:
It took me a few moments to understand what the bugfix is here -- because initially, this diff seems to simply rename two variables.  The fix relates to an outer, existing `item` variable name that was being shadowed/clobbered by the local loop variable (`const <x> of ...`).  The fix is to use a distinct variable name so that the outer object remains accessible and allows the `item[...]` syntax to retrieve a value from it.